tests/src/test/java/org/eclipse/hono/tests/registry/CorsIT.java

```diff
@@ -0,0 +1,224 @@
+/*******************************************************************************
+ * Copyright (c) 2016, 2019 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package org.eclipse.hono.tests.registry;
+
+import io.vertx.core.MultiMap;
+import io.vertx.core.Vertx;
+import io.vertx.core.http.HttpHeaders;
+import io.vertx.core.http.HttpMethod;
+import io.vertx.ext.unit.TestContext;
+import io.vertx.ext.unit.junit.VertxUnitRunner;
+import static org.eclipse.hono.tests.DeviceRegistryHttpClient.TEMPLATE_URI_CREDENTIALS_BY_DEVICE;
```

Comment:
the imports are out of order and
IMHO we should not statically import constants but only import the defining class and the then refer to the constants in a qualified way. Otherwise, it is very hard to see where the constant comes from when reading the code ...

Reply:
Changed it. BTW. this is not done consistently throughout codebase, so maybe we should add checkstyle rule if that's what we want to do?

Reply:
This is at least what I would like to do. Not sure if there is a corresponding Checkstyle check, though ...